sdk/containerservice/arm-containerservice/package.json

```diff
@@ -51,7 +52,7 @@
     "dist/",
     "README.md",
     "LICENSE",
-    "review/",
+    "review/*",
```

Comment:
We didn't remove the dependency of api extractor and changelog in the codegen and release tool?

Reply:
yes